packages/dd-agent/packaging

```diff
@@ -39,20 +39,20 @@ echo "extracting package"
 if [ $OS == "Debian" ]; then
   dpkg -x dd-agent/datadog-agent_${DD_AGENT_VERSION}-1_amd64.deb ./extracted-agent
 
-  cp -R ./extracted-agent/opt/datadog-agent/* ${BOSH_INSTALL_TARGET}
-  cp -R ./extracted-agent/etc/datadog-agent ${BOSH_INSTALL_TARGET}/etc
+  cp -Rp ./extracted-agent/opt/datadog-agent/* ${BOSH_INSTALL_TARGET}
```

Comment:
```suggestion
  chown root:root ./extracted-agent/opt/datadog-agent/embedded/bin/system-probe
  chown -R root:root ./extracted-agent/opt/datadog-agent/embedded/share/system-probe/ebpf
  cp -Rp ./extracted-agent/opt/datadog-agent/* ${BOSH_INSTALL_TARGET}
```

Reply:
This comes from the `postinst` script for the `deb` package: https://github.com/DataDog/datadog-agent/blob/db3d46ac9aef252a90345bb7c3fa9f040b69df71/omnibus/package-scripts/agent-deb/postinst#L94-L96